docs/content/integrations/airlift/operator-migration/overview.mdx

```diff
@@ -9,4 +9,12 @@ This page contains a collection of reference materials for migrating usage of co
     title="Bash Operator for dbt"
     href="/integrations/airlift/operator-migration/bash-operator-dbt"
   ></ArticleListItem>
+  <ArticleListItem
+    title="Kubernetes Pod Operator"
+    href="/integrations/airlift/operator-migration/kubernetes-pod-operator"
+  ></ArticleListItem>
+  <ArticleListItem
+    title="Python Operator"
+    href="/integrations/airlift/operator-migration/python-operator"
+  ></ArticleListItem>
```

Comment:
Should this be added here?

Reply:
ah - I think this got added from the downstream somehow. Will delete.